modules/ROOT/pages/advanced/creating-a-plugin.adoc

```diff
@@ -88,14 +88,14 @@ Due to the range of browser APIs used by {productname}; when testing {productnam
 
 == Language localization
 
-If a custom plugin includes any custom UI created using {productname}'s link:{baseurl}/ui-components[UI APIs], then it may require localization.
+If a custom plugin includes any custom UI created using {productname}'s xref:ui-components.adoc[UI APIs], then it may require localization.
```

Comment:
This xref doesn't resolve... I believe it should be this:
```suggestion
If a custom plugin includes any custom UI created using {productname}'s xref:ui-components/index.adoc[UI APIs], then it may require localization.
```

Reply:
done